srcpkgs/pounce/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'pounce'
 pkgname=pounce
-version=2.3
-revision=2
-build_style=gnu-configure
+version=2.4
+revision=1
+build_style=configure
```

Comment:
Please explain the switch from `gnu-configure` in the commit message. AFAIK June's stuff carried "autoconf compatible" configure scripts.

Reply:
I didn't look into it enough, sorry, reverted.